### PR TITLE
docs: Updates on Setup and Demo documentation

### PR DESCRIPTION
**Pull request by rodrigobersa**
## Description of changes

* Update formatting and setup steps.
* Moved all the testing and validation from `docs/Setup.md` to `docs/Demo.md`, instructing to follow the steps on that page.
* Moved motivation to the main README.md file.
* Moved the RBAC Converter to it's own section.
* Updated the Docs table on the main README.md

### CODE REVIEW

File: docs/Demo.md [hunk 1]
```diff
@@ -408,21 +374,42 @@ forbid (
...
+KUBECONFIG=./mount/sample-user-kubeconfig.yaml kubectl get cm # denied
+KUBECONFIG=./mount/sample-user-kubeconfig.yaml kubectl label cm/other-config stage=test # denied
+```
 
-KUBECONFIG=./mount/sample-user-kubeconfig.yaml kubectl get cm -l owner=sample-user --show-labels
-cat << EOF > sample-config.yaml
+Try to create a new ConfigMap.
+
+```bash
+KUBECONFIG=./mount/sample-user-kubeconfig.yaml kubectl create cm sample-config --from-literal=k1=v1 # allowed
```
Comment (micahhausler) [inline docs/Demo.md:406]: This should be denied, the request is missing the required labels

File: docs/ConvertRBAC.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# Convert RBAC policies
+
+There's an RBAC converter that works on ClusterRoleBindings or RoleBindings.
```
Comment (micahhausler) [inline docs/ConvertRBAC.md:3]: I'll follow up with a PR to add some explanation (like how plain RBAC is going to be faster by nature of not needing an external webhook call, but that this is to facilitate migration of policies to be more restrictive)

**Review by micahhausler — CHANGES_REQUESTED** [review]
One minor change, LGTM otherwise

File: docs/Demo.md (hunk 1 above)
Comment (rodrigobersa) [reply docs/Demo.md]: Fixed as we spoke :)

File: docs/ConvertRBAC.md (hunk 2 above)
Comment (rodrigobersa) [reply docs/ConvertRBAC.md:3]: Cool, let me know if you need review on that before merging.

**Review by micahhausler — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**